content/backstage/plugins/firebase-functions.md

```diff
@@ -26,17 +26,52 @@ gettingStarted:
     language: 'ts'
     code: |
       // packages/app/src/plugins.ts
```

Comment:
Has this plugin actually been updated to use the composability API? I ask, because this step of adding the plugin to the `src/plugins.ts` page seems to not be needed in the other plugins I've looked at which have been migrated.